pkgs/applications/misc/kitty/default.nix

```diff
@@ -19,12 +32,32 @@ buildPythonApplication rec {
   };
 
   buildInputs = [
-    fontconfig glfw ncurses libunistring harfbuzz libX11
+    ncurses harfbuzz
+  ] ++ stdenv.lib.optionals stdenv.isDarwin [
+    Cocoa
+    CoreGraphics
+    Foundation
+    IOKit
+    Kernel
+    OpenGL
+    cf-private
+    libpng
```

Comment:
This is weird that darwin needs it but not linux.

Reply:
Yeah, I thought that was weird too. (Same with zlib.)

In this commit, I tried to avoid changing how kitty is built on Linux.